src/gui/sdl_mapper.cpp

```diff
@@ -1646,6 +1643,7 @@ class CBindButton final : public CClickableTextButton {
 			}
 			if (mapper.abindit!=mapper.aevent->bindlist.end()) SetActiveBind(*(mapper.abindit));
 			else SetActiveBind(nullptr);
```

Comment:
Suggest fixing up the formatting on these 2 lines.  This is our formatting style for if/else statements (for new code, we still have legacy code that doesn't follow our style)  I know you didn't write this but you added a line after this single-line else so it would be nice to clean this up for readability.

```cpp
if (mapper.abindit != mapper.aevent->bindlist.end()) {
    SetActiveBind(*(mapper.abindit));
} else {
    SetActiveBind(nullptr);
}

Reply:
Another formatting tip @arifogel, we use Clang Format, there is a `.clang-format` config at the root of the repo. All popular editors and IDEs can be configured or have plugin supportfor Clang Format. I personally use [vim-clang-format](https://github.com/rhysd/vim-clang-format).

Reply:
done, just rebased again

Reply:
@johnnovak I am running `./scripts/format-commit.sh` on each commit, but AFAICT it only touches changed lines. I can set my IDE to clang-format automatically, but I would still only want it to hit changed lines so my PRs are legible. So it would not have been caught because:

> I know you didn't write this but you added a line after this single-line else so it would be nice to clean this up for readability.

Reply:
That script will usually look a couple lines above and below changed lines and format them as well but it's not perfect.  Sometimes it will even completely butcher formatting with certain patterns so it's worth re-checking manually.  Most of the time it works alright though.

Reply:
Makes sense @arifogel. Yeah, sometimes it needs a little manual help. It's handy to have it configured for your IDE too so you can reformat only the selected lines. I use that all the time.